app/app.go

```diff
@@ -1045,11 +1045,11 @@ func (app *App) RegisterStreamingServices(appOpts servertypes.AppOptions) {
 			logLevel := cast.ToString(appOpts.Get(flags.FlagLogLevel))
 			plugin, err := streaming.NewStreamingPlugin(pluginName, logLevel)
 			if err != nil {
-				app.Logger().Error("failed to load streaming plugin: %s", err)
+				app.Logger().Error("failed to load streaming plugin", "error", err)
```

Comment:
For reference on the weird argument syntax here:
```
// Only map[string]interface{} and []interface{} are accepted. []interface{} must
// alternate string keys and arbitrary values, and extraneous ones are ignored.
```
The method is going to use the pair of arguments as key/values.